index.html

```diff
@@ -48,6 +45,124 @@ <h2>Let's Connect</h2>
+                    Stories</h1>
+            
+                    <ul class="field">
+                        <li class="cano">CANOPY</li>
+                        <li class="tag">&#8226; Back End Dev </li>
+                        <li class="tag">&#8226; 2015</li>
+                    </ul>
+                <p class="instruction"> A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.</p>
+                <ul class="trainning">
+                    <li class="other-skills">html</li>
+                    <li class="other-skills">css</li>
+                    <li class="other-skills">javascript</li>
+                </ul>
+                <button class="card-btn">See Project</button>
+            </div>
+            </section>
+            
+            <!--about-me-section-->
+            <section class="about-myself">
+                <div class="content">
+                <h1 class="myself">About Myself</h1>
+                <p class="about-para"> Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            
+                <h2 class="about-sub-head">Let's Connect</h2>
+                <div class="social-media">    
+                    <ul>
+                        <li><a href="#"><img src="icon/twitter.svg" alt="twitter"></a></li>
+                        <li><a href="#"><img src="icon/linkdn.png" alt="linkdn-icon"></a></li>
+                        <li><a href="#"><img src="icon/m.svg" alt="media-con"></a></li>
+                        <li><a href="#"><img src="icon/hand.svg"alt="github-con"></a></li>
+                        <li><a href="#"><img src="icon/git.svg"></a></li>
+                    </ul>
+                </div>
+                     <button class="resume-btn">Get My resume</button>
+            </div>
+            <div class="language-header">
+                <h3 class="language-heading">Languages<i class="fa fa-angle-down"></i></i></h3>
+              </div>
+              <ul class="list-language">
+                  <li class="for-web"><img src="icon/js.png" alt="">JavaSript</li>
+                  <li class="for-web"><img src="icon/html.png" alt="">HTML</li>
+                  <li class="for-web"><img src="icon/css.png" alt="">CSS</li>
+              </ul> 
+              <ul class="frame-work">
+                  <li class="skill">
+                      <h3 class="first-title">Frameworks<i class="fa fa-angle-right" ></i></h3>
+                      <hr class="line">  
+                  </li>
+                  <li class="skills">
+                      <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3> 
+                      <hr class="line">
+                  </li>
+              </ul>
```

Comment:
- [ ] You got the idea of using lists to display the elements of the page, but in this case, a better approach would be to have an outer `ul` that englobe the 'Languages', 'Frameworks', and 'Skills' as list items, and another `ul` inside the 'Languages' item that englobe all the languages(JS, HTML and CSS). Kindly make those changes for a better HTML semantic. 
 - [ ] Also, the font-family was not correctly applied to some items in the list like the languages and the 'Framework' item. Kindly add the font-family to them so it can look more professional and similar to the Figma template.
 - Your design:
 - 
<img width="446" alt="image" src="https://user-images.githubusercontent.com/61919617/227321978-94dcb06c-070f-4a3d-9f05-e2c4bdce9d0e.png">